x-pack/legacy/plugins/uptime/server/lib/adapters/monitor_states/elasticsearch_monitor_states_adapter.ts

```diff
@@ -29,11 +29,121 @@ interface LegacyMonitorStatesQueryResult {
   afterKey: any | null;
 }
 
+interface LegacyMonitorStatesRecentCheckGroupsQueryResult {
+  checkGroups: string[];
+  afterKey: any | null;
+}
+
+const legacyStatesQuerySize = 50;
```

Comment:
Can we move this to the shared constants directory? `~/uptime/common/constants/query.ts` is probably the best place.

Reply:
++